editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -4718,6 +4718,124 @@ void SpatialEditor::_refresh_menu_icons() {
 	tool_button[TOOL_UNLOCK_SELECTED]->set_visible(all_locked);
 }
 
+template <typename T>
+Set<T *> _get_child_nodes(Node *parent_node) {
+	Set<T *> nodes = Set<T *>();
+	T *node = Node::cast_to<T>(parent_node);
+	if (node) {
+		nodes.insert(node);
+	}
+
+	for (int i = 0; i < parent_node->get_child_count(); i++) {
+		Node *child_node = parent_node->get_child(i);
+		Set<T *> child_nodes = _get_child_nodes<T>(child_node);
+		for (typename Set<T *>::Element *I = child_nodes.front(); I;
```

Comment:
Please don't wrap forcefully when it makes things less readable, like here (several other occurrences). In general we don't have max char limit because the wrap it imposes it almost always bad for readability.

Reply:
Must a clang format misconfiguration on my end